runtime/runtime/src/lib.rs

```diff
@@ -1074,10 +1074,9 @@ impl Runtime {
         )?;
 
         state_update.commit(StateChangeCause::UpdatedDelayedReceipts);
-        // TODO(#2327): Avoid cloning.
-        let state_changes = state_update.committed_updates_per_cause().clone();
```

Comment:
It seems it was the only place we used this method, so we should also remove its implementation.